__tests__/app.test.js

```diff
@@ -0,0 +1,32 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection.js');
+const seed = require('../db/seeds/seed');
+const testData = require('../db/data/test-data');
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/categories', () => {
+    test('returns a correctly formatted category object', () => {
+        return request(app).get('/api/categories').expect(200).then((response) => {
+            expect(response.body.categories[0]).toMatchObject({
```

Comment:
I would recommend doing a forEach here rather than checking [0] in order to ensure they are all identical and it is not just the first element. You would then also need to check the length of the array.